canonical-websites/www.ubuntu.com
# Add the 18.10 takeover

## Done
Created the 18.10 takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the takeover matches the [design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/159) and [copy doc](https://docs.google.com/document/d/18s-sTyuuYZP7xrpn7MtN5ABScfv0QVOCdRbEGZ44cg4/edit)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4037
